bundles/CoreBundle/src/Command/Bundle/InstallCommand.php

```diff
@@ -49,6 +49,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $bundle = $this->getBundle();
 
+        if ($this->bundleManager->isInstalled($bundle)) {
+            $this->io->success(sprintf('Bundle "%s" is already installed', $bundle->getName()));
+            return 0;
```

Comment:
I think we should start using `Command::SUCCESS`, etc. in Pimcore.

See: https://symfony.com/blog/new-in-symfony-5-1-misc-improvements-part-1#added-constants-for-command-exit-codes

Reply:
Thanks @jdreesen, I've adjusted based on your suggestion